src/main/kotlin/tech/relaycorp/poweb/PoWebClient.kt

```diff
@@ -105,7 +105,7 @@ public class PoWebClient internal constructor(
 
         requireContentType(PNRA_CONTENT_TYPE, response.contentType())
 
-        val authorizationSerialized = response.content.toByteArray()
+        val authorizationSerialized = response.readBytes()
```

Comment:
In the app projects I have the helper:
`fun InputStream.readBytesAndClose() = readBytes().also { close() }` to make sure we read and close.

Reply:
Thanks! That'd explain why this change didn't fix it. I'll try that.